ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -78,12 +87,18 @@ function ExceptionRequestDetailsPage() {
         data: vulnerabilityException,
         loading,
         error,
+        refetch,
     } = useRestQuery(vulnerabilityExceptionByIdFn);
 
     function handleTabClick(event, value) {
         setSelectedTab(value);
     }
 
+    function onApprovalSuccess(exception: VulnerabilityException) {
```

Comment:
As part of the approval success, can we invalidate the workload CVE queries via the new hook? (Or maybe as a follow up, since I know the ticket for that is separate.)

Reply:
Yea, we can do that in a separate PR 👍🏼